index.html

```diff
@@ -430,7 +430,175 @@ <h2>
         issuance protocol is identified by a [=digital credential/protocol
         identifier=]. See also section [[[#protocol-registry]]].
       </dd>
+      <dt>
+        Digital credential coordinator
+      </dt>
+      <dd>
+        See [=credential request coordinator=].
+      </dd>
+      <dt>
+        Digital credential client
+      </dt>
+      <dd>
+        See [=credential request client=].
+      </dd>
     </dl><!--
+    // MARK: Credential Request Coordinator
+    -->
+    <h2>
+      Credential Request Coordinator
+    </h2>
+    <p>
+      The <dfn data-local-lt="coordinator">credential request coordinator</dfn>
+      is a user-agent-defined component responsible for mediating [=digital
+      credential=] interactions within a given [=top-level traversable=]. It
+      ensures that only one interaction is active at any time across [=child
+      navigables=], and it manages the state transitions between different
+      [=credential request coordinator/interaction states=].
+    </p>
+    <p>
+      Each [=top-level traversable=] has a single associated [=credential
+      request coordinator=].
+    </p>
+    <p>
+      The coordinator manages the lifecycle of the current interaction's
+      {{Promise}}, including its resolution or rejection and integrates with an
+      {{AbortSignal}} to allow for both user-initiated and programmatic
+      cancellation.
+    </p>
+    <p>
+      A [=credential request coordinator=] has an associated [=credential
+      request client=], which it interacts with.
+    </p>
+    <h3>
+      Interaction states
+    </h3>
+    <p>
+      The [=credential request coordinator=] has a finite set of
+      <dfn data-dfn-for="credential request coordinator">interaction
+      states</dfn>, which are used to manage the lifecycle of request for a
+      digital credential, be it either a [=digital credential/presentation
+      request=] or an [=digital credential/issuance request=]. The states are:
+    </p>
+    <dl>
+      <dt>
+        <dfn data-dfn-for="credential request coordinator">idle</dfn>:
+      </dt>
+      <dd>
+        No credential interaction is currently in progress.
+      </dd>
+      <dt>
+        <dfn data-dfn-for="credential request coordinator">requesting</dfn>:
+      </dt>
+      <dd>
+        A credential interaction is currently in progress and the user
+        interface is being presented to the user.
+      </dd>
+      <dt>
+        <dfn data-dfn-for="credential request coordinator">aborting</dfn>:
+      </dt>
+      <dd>
+        A credential interaction is being canceled, either because of an error,
+        user cancellation, or a [=AbortController/signal abort=]. The
```

Comment:
```suggestion
        A credential interaction is being canceled, because of an error,
        a user cancellation, or a [=AbortController/signal abort=]. The
```